### ISSUE TITLE
PHP 7 Errors are caught incorrectly

### ISSUE THREAD

**Issue by sandfoxme** [rollbar/rollbar-php#96]
```
ERROR: Uncaught TypeError: Argument 2 passed to RollbarException::__construct() 
must be an instance of Exception, instance of Error given, 
called in .../vendor/rollbar/rollbar/src/rollbar.php on line 345 
and defined in .../vendor/rollbar/rollbar/src/rollbar.php:91
```

One type hint was not removed but probably it is better to use ``class_alias()`` instead of define, it will also help to restore class hinting removed here https://github.com/rollbar/rollbar-php/commit/7b7cae9777ee794935b5a1b778fab6d128b80629

**Comment by sandfoxme** [comment]
I will try to make a PR on weekend unless there's someone faster :)

**Comment by aensley** [comment]
I see this was fixed in [0.18.2](https://github.com/rollbar/rollbar-php/releases/tag/v0.18.2). It should probably be closed.

**Comment by sandfoxme** [comment]
It seems it works
